.gitignore

```diff
@@ -64,3 +64,6 @@ override.tf.json
 # Ignore CLI configuration files
 .terraformrc
 terraform.rc
+/deploy/serverless-network/lambda-layers/all-dep/python/
```

Comment:
Is there a cleaner way to define these in the gitignore? Seems a bit verbose.

Reply:
Done.